spec/requests/api/v1/teams_request_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "swagger_helper"
+
+RSpec.describe "Teams authentified API", swagger_doc: "v1/api.json" do
+  with_examples
+
+  path "/api/v1/teams" do
+    get "Lister les équipes" do
+      with_authentication
+
+      tags "Teams"
+      produces "application/json"
+      operationId "getTeams"
+      description "Renvoie les équipes des territoires de l'agent authentifié."
```

Comment:
nitpick mais on dirait que ça va retourner les équipes desquelles fait partie l’agent

```suggestion
      description "Renvoie toutes les équipes de tous les territoires auquel l'agent authentifié a accès."
```